chef/roles/ceph-mon.rb

```diff
@@ -1,5 +1,5 @@
 name "ceph-mon"
 description "Ceph Monitor"
 run_list(
-        "recipe[ceph::mon]"
+        "recipe[ceph::role_ceph_mon]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.